docs/spfx/compatibility.md

```diff
@@ -36,22 +36,22 @@ The following table lists SharePoint Framework and compatible versions of common
 
 |SPFx|Node.js|NPM|TypeScript|React
 |---|---|---|---|---|
-|[1.10.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.10-release-notes)|LTS v10.x|v5, v6|v3.3|v16.8.5|
-|[1.9.1](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.9.1-release-notes)|LTS v10.x|v5, v6|v2.9|v16.8.5|
-|[1.9.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.9-release-notes) | LTS 8.x, LTS 10.x |v5, v6 |v2.9|v16.8.5|
-|[1.8.2](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.8.2-release-notes)| LTS 8.x, LTS 10.x | v5, v6 |v2.9|v16.7.0|
-|[1.8.1](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.8.1-release-notes)| LTS 8.x | v5, v6|v2.7, v2.9, v3.x|v16.7.0|
-|[1.8.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.8-release-notes)| LTS 8.x | v5, v6|v2.7, v2.9, v3.x|v16.7.0|
-|[1.7.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.7.1)| LTS 8.x | v5, v6 |v2.4|v16.3.2|
-|[1.7.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.7-release-notes)| LTS 8.x | v5, v6 |v2.4|v16.3.2|
-|[1.6.0](https://github.com/SharePoint/sp-dev-docs/wiki/SharePoint-Framework-v1.6-release-notes)|LTS 6.x,  LTS 8.x | v3 (w/ Node.js 6.x),<br/> v5 (w/ Node.js 8.x)|v2.4|v15.x|
-|[1.5.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.5.1)|LTS 6.x,  LTS 8.x | v3 (w/ Node.js 6.x),<br/> v5 (w/ Node.js 8.x)|v2.4|v15.x|
-|[1.5.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SharePoint-Framework-Package-v1.5)|LTS 6.x,  LTS 8.x | v3 (w/ Node.js 6.x),<br/> v5 (w/ Node.js 8.x)|v2.4|v15.x|
-|[1.4.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.4.1)|LTS 6.x,  LTS 8.x | v3, v4 |v2.4|v15.x|
-|[1.4.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.4)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.2.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes---Extensions-RC-Drop-1.2)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.1.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes---Extensions-Dev-Preview-Drop-1)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.0.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-GA)|LTS 6.x| v3 |v2.4|v15.x|
+|[1.11.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.11.0)|LTS v10.x|v5, v6|v3.3|v16.8.5|
```

Comment:
Please remove the locale from the URL... let MSDocs redirect based on user agent (per MSDocs)

Reply:
Argh! Those darned locales get me every time. I've made the changes as requested